HoloLensCommander/HoloLensCommander/Utilities/Settings.cs

```diff
@@ -71,28 +92,49 @@ public float HeartbeatInterval
         /// Initializes a new instance of the <see cref="Settings" /> class.
         /// </summary>
         public Settings() :
-            this(DefaultHeartbeatInterval)
+            this(
+            DefaultHeartbeatInterval)
         { }
 
         /// <summary>
         /// Initializes a new instance of the <see cref="Settings" /> class.
         /// </summary>
         /// <param name="interval">Seconds between heartbeat checks.</param>
-        public Settings(float interval) : 
-            this(false, interval)
+        public Settings(float interval) :  
+            this(
+            false, 
```

Comment:
I know the other constructor is just a few lines later, but it would still be nice to have the variable names commented in your base initializer since there are so many.